index.js

```diff
@@ -96,7 +96,7 @@ module.exports = {
       this.app = app = app.app
     }
 
-    this._super.included(app)
+    this._super.included.apply(this, app)
```

Comment:
🤦‍♂️ that is probably the one thing I didn't try. Now I know!

Reply:
ya, the `this._super.ANYTHING_HERE` methods are all essentially unbound (they are actually bound to the prototype, through "standard javascript `this` rules"), so you _always_ have to enforce the correct `this` binding via `this._super.ANYTHING_HERE.apply(this, arguments)`.